filament/src/fg/FrameGraph.h

```diff
@@ -527,7 +527,7 @@ class FrameGraph {
 
 template<typename Data, typename Setup, typename Execute>
 FrameGraphPass<Data>& FrameGraph::addPass(char const* name, Setup setup, Execute&& execute) {
-    static_assert(sizeof(Execute) < 1024, "Execute() lambda is capturing too much data.");
+    static_assert(sizeof(Execute) < 2048, "Execute() lambda is capturing too much data.");
```

Comment:
ouch. why was this needed? Which pass has the issue?

Reply:
It's the `prepareShadowPass`, which now requires 1064 bytes. Maybe we can raise it to something like 1128.

Reply:
Ah yes because CameraInfo is huge now. Okay. I'm fine with 2048. This was there to catch unintended captures, at the time 1KiB seemed reasonable. Because this is deferred we need to capture by copy.